.github/workflows/release.yaml

```diff
@@ -51,80 +48,56 @@ jobs:
           packages-dir: python/dist/
 
   publish-plugins:
-    needs: publish-core
     name: Create Plugin Releases
     defaults:
       run:
         working-directory: ./python
-    strategy:
-      max-parallel: 1
-      matrix:
-        os: [ubuntu-latest]
-        python-version: ["3.10"]
-        package:
-          [
-            autogen,
-            camel,
-            claude,
-            crew_ai,
-            griptape,
-            julep,
-            langchain,
-            llamaindex,
-            lyzr,
-            openai,
-            praisonai,
-            langgraph,
-            phidata,
-            google,
-          ]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Build ${{ matrix.package }}
+          python-version: "3.10"
+      - name: Build plugins
         run: |
-          pip install twine build
-
-          # Build dist
-          cd plugins/${{ matrix.package }}
-          python -m build
-      - name: Publish ${{ matrix.package }} to PyPI
+          pip install build twine
+
+          plugins="autogen camel claude crew_ai griptape julep langchain llamaindex lyzr openai praisonai langgraph phidata google"
+          mkdir plugins_dist
+          for plugin in $plugins; do
+            cd plugins/$plugin
+            python -m build
+            mv dist/* ../../plugins_dist/
+            cd ../..
+          done
+      - name: Publish plugins to PyPI
+        if: github.event_name == 'release'
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: ${{ secrets.PYPI_USERNAME }}
           password: ${{ secrets.PYPI_PASSWORD }}
           skip-existing: true
```

Comment:
The plugin build process has changed to build all plugins in a loop instead of using a matrix. While functionally equivalent, this removes the parallel build capability of the matrix, potentially increasing build time significantly. To restore parallel builds, revert to the original matrix-based approach (lines 59-80).